plugin/al/mayatest/AL/maya/test/CMakeLists.txt

```diff
@@ -1,66 +1,61 @@
 
 ####################################################################################################
-# Setup 
+# Setup
 ####################################################################################################
 find_package(GTest REQUIRED)
 
 set(MAYA_TEST_LIBRARY_NAME "AL_MayaTest")
-set(MAYA_TEST_LIBRARY_LOCATION ${AL_INSTALL_PREFIX}/lib) 
+set(MAYA_TEST_LIBRARY_LOCATION ${AL_INSTALL_PREFIX}/lib)
 
 ####################################################################################################
-# Source 
+# Source
 ####################################################################################################
+add_library(${MAYA_TEST_LIBRARY_NAME} SHARED)
 
-list(APPEND maya_test_headers
-    Api.h
-    testHelpers.h
-    testHarness.h
-)
-
-list(APPEND maya_test_source
+target_sources(${MAYA_TEST_LIBRARY_NAME}
+  PRIVATE
     testHelpers.cpp
     testHarness.cpp
 )
```

Comment:
Use `target_sources` which is a more concise approach for adding sources to a target. We encourage using cmake `target_` commands as much as possible.